README.md

```diff
@@ -162,9 +162,13 @@ ln -s /usr/local/opt/openssl@3/bin/openssl /usr/local/bin/openssl
 ```
 
 ## Configure cf-k8s-controllers
-Configuration file for cf-k8s-controllers is at `config/base/controllersconfig/cf_k8s_controllers_k8s.yaml`
+Configuration file for cf-k8s-controllers is at `controllers/config/base/controllersconfig/cf_k8s_controllers_k8s.yaml`
```

Comment:
Thanks for updating this path in the README, looks like I missed one when I was reviewing the workloads tls.